### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MSerg2306**

### CODE REVIEW

File: AirportPanel/Program.cs [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+            s += "| " + String.Format(Airline + new string(' ', 15 - Airline.ToString("g").Length)) + " ";
+            s += "| " + Program.CentrStr(Terminal.ToString("g"), 7);
+            s += "| " + String.Format("{0:d3}", Gate) + " ";
+            s += "| " + String.Format(Status + new string(' ', 10 - Status.ToString("g").Length)) + " ";
+            s += "| ";
+
+            Console.Write(s);
+        }
+    }
+    class Program
```
Comment (Ruslan4) [inline AirportPanel/Program.cs:74]: Bad practic write long class. You must remember about [KISS](https://en.wikipedia.org/wiki/KISS_principle), an acronym for "keep it simple, stupid"

File: AirportPanel/Program.cs [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.SetCursorPosition(0, 40);
+            Console.Write(new string(' ', 164));
+            Console.SetCursorPosition(0, 40);
+            Console.WriteLine("MENU: 1-ViewList 2-AddFlight 3-DeliteFlight 4-EditFlight 5-FindFlight 6-SpecialFunction Q-Quit");
+
+            return flight;
+        }
+        public static void TableHead()
```
Comment (Ruslan4) [inline AirportPanel/Program.cs:199]: Remember about code convension.  (Formating. Empty string before name method)

File: AirportPanel/Program.cs [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+            }
+            return (result);
+        }
+        public static bool InputInOut(int left, int top)
+        {
+            Console.SetCursorPosition(0, 40);
+            Console.Write(new string(' ', 164));
+            Console.SetCursorPosition(left, top);
+            int i = -1;
+            string s = null;
```
Comment (Ruslan4) [inline AirportPanel/Program.cs:242]: Use more informative variable name

File: AirportPanel/Program.cs [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+                else
+                {
+                    Console.SetCursorPosition(0, 40);
+                    Console.BackgroundColor = ConsoleColor.Black;
+                    Console.Write("Please enter correctly. " + s);
+                    Console.SetCursorPosition(left, top);
+                }
+            }
+            return (AirlineName)number;
+
```
Comment (Ruslan4) [inline AirportPanel/Program.cs:414]: why empty string?)

File: Work14/Program.cs [hunk 5]
```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Work4
+{
+    enum NumberToWord
```
Comment (Ruslan4) [inline Work14/Program.cs:9]: use EN name!

File: Work14/Program.cs [hunk 6]
```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Work4
```
Comment (Ruslan4) [inline Work14/Program.cs:7]: Very long namespace. All code in one file(

File: Work14/Program.cs [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+                    }
+                    else
+                    {
+                        Console.Write("Вы ввели что-то не то...");
+                    }
+                }
+            }
+            return cs;
+        }
+        public static void GameSpecification(string name)
```
Comment (Ruslan4) [inline Work14/Program.cs:91]:
```suggestion
        public static void GameSpecification(string gameName)
```

File: Work14/Program.cs [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+        }
+    }
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[,] GameHistory = new int[100, 6];
+            Gamer gamer = new Gamer(100);
+            Console.Write("Представтесь, пожалуйста... - ");
+            gamer.Name = Console.ReadLine();
```
Comment (Ruslan4) [inline Work14/Program.cs:144]: What hapends when user write wrong gamer name?

**Review by Ruslan4 — CHANGES_REQUESTED** [review]
Fix it before merging.

File: AirportPanel/Program.cs (hunk 1 above)
Comment (MSerg2306) [reply AirportPanel/Program.cs:74]: how to simplify the withdrawal of the flight string? This is a static method and I brought it to a separate ...

File: AirportPanel/Program.cs (hunk 2 above)
Comment (MSerg2306) [reply AirportPanel/Program.cs:199]: I agree, it was necessary to work through constants (40 and 160) ...,

automate calculation 61 (Airline = InputAirline (61, 6),
+ Terminal = InputTerminal (61, 7),
+ Gate = InputNumberGate (61, 8),
+ Status = InputStatus (61, 9), is through the structure of the header and its method for calculating the maximum length?

File: AirportPanel/Program.cs (hunk 3 above)
Comment (MSerg2306) [reply AirportPanel/Program.cs:242]: agree, "c" and "l" needed to be described in more detail, for example, Text and CellLength

File: AirportPanel/Program.cs (hunk 4 above)
Comment (MSerg2306) [reply AirportPanel/Program.cs:414]: did you find a bug? I always saw the inscription

File: Work14/Program.cs (hunk 5 above)
Comment (MSerg2306) [reply Work14/Program.cs:9]: I trained on my daughter (8 years old), she played for 2 days, won 1000 once), ok

File: Work14/Program.cs (hunk 6 above)
Comment (MSerg2306) [reply Work14/Program.cs:7]: we did not learn classes yet and I realized within the limits of the material studied, but yes, I agree, it’s not right

File: Work14/Program.cs (hunk 7 above)
Comment (MSerg2306) [reply Work14/Program.cs:91]: gamerName, I agree

File: Work14/Program.cs (hunk 8 above)
Comment (MSerg2306) [reply Work14/Program.cs:144]: just started writing, not yet warmed up ... (
nothing, there will be an empty name, it string allows null

File: AirportPanel/Program.cs (hunk 1 above)
Comment (Ruslan4) [reply AirportPanel/Program.cs:74]: This solution is also good, but struct Flight better to put in a separate file.

File: AirportPanel/Program.cs (hunk 4 above)
Comment (Ruslan4) [reply AirportPanel/Program.cs:414]: return (AirlineName)number;
     <empty lines>
}

It is necessary to remove empty lines in the code, just for more convenient code maintenance)

**Pull request merged**